ui/src/views/Rules/Rule/index.jsx

```diff
@@ -391,34 +393,50 @@ function Rule({ isNewRule, user, ...props }) {
   }, [ruleId, scId]);
 
   useEffect(() => {
-    fetchRequiredSignoffs(OBJECT_NAMES.PRODUCT_REQUIRED_SIGNOFF).then(
-      rsResponse => {
-        const requiredSignoffs = rsResponse.data.data.required_signoffs;
-
-        if (rule.product) {
-          const matchingRs = requiredSignoffs.filter(rs =>
-            ruleMatchesRequiredSignoff(rule, rs)
-          );
-
-          if (!requiredSignoffs.length || !matchingRs.length) {
-            setSignoffSummary(' Nobody');
+    const rs =
+      requiredSignoffs.data && requiredSignoffs.data.data.required_signoffs;
+
+    if (rs) {
```

Comment:
Although it's unlikely to be the case, we should call `setSignoffSummary` if we don't enter this if block - otherwise it will be `null`.

Reply:
Ok, lemme work on this.